### PR TITLE
Add locate service

### PR DESCRIPTION
**Pull request by dirkschumacher**
Sometimes you would like to know where the nearest market is.
One solution is to locate your position. Therefor I added a little leaflet plugin to do that.

bildschirmfoto 2016-04-18 um 19 06 35

### CODE REVIEW

**Comment by johnjohndoe** [comment]
@tursics Please review.

**Comment by tursics** [comment]
Super, nice! Please, can you change the z-index of the blue bullet so that the market pins do not hide the bullet?

**Comment by dirkschumacher** [comment]
@tursics no idea how to do this :)

**Comment by tursics** [comment]
You must use different layers for both kind of objects and use layerName.setZIndex(7)

**Comment by dirkschumacher** [comment]
Would you mind pasting some sample code? I still have no idea how to do this. Thanks a lot ! :)

**Comment by dirkschumacher** [comment]
Impossibru! I don't know. Can't get it to work. I would suggest, :shipit: or someone else looks into this.

File: index.html [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-        <link rel="stylesheet" href="lib/leaflet.awesome-markers/leaflet.awesome-markers.css"/>
-        <link rel="stylesheet" href="lib/leaflet.locatecontrol/L.Control.Locate.min.css"/>
-        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>
-        <link href="lib/select2/select2.min.css" rel="stylesheet" />
+        <link rel="stylesheet" href="lib/normalize/normalize.css">
+        <link rel="stylesheet" href="lib/leaflet/leaflet.css">
+        <link rel="stylesheet" href="lib/leaflet.awesome-markers/leaflet.awesome-markers.css">
+        <link rel="stylesheet" href="lib/leaflet.locatecontrol/L.Control.Locate.min.css">
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+        <link href="lib/select2/select2.min.css" rel="stylesheet">
```
Comment (johnjohndoe) [inline index.html]: @dirkschumacher What did you use to lint the HTML? I tried [html-tidy5](https://github.com/htacg/tidy-html5) with [SublimeLinter-html-tidy](https://github.com/SublimeLinter/SublimeLinter-html-tidy). I did not get any warning nor error though.

Comment (dirkschumacher) [inline index.html]: What do you mean? In html5 it is not needed. I did not use any linter. http://stackoverflow.com/a/7366380

**Comment by dirkschumacher** [comment]
@johnjohndoe rebased

**Comment by johnjohndoe** [comment]
Looks good to me. Merged. Pushed. Thank you :neckbeard:

**Pull request merged**